ee/vellum_ee/workflows/tests/test_display_meta.py

```diff
@@ -102,3 +103,38 @@ class MyCustomWorkflowDisplay(BaseWorkflowDisplay[MyCustomWorkflow]):
     assert display_meta.workflow_outputs == {
         "answer": workflow_output_id,
     }
+
+
+def test_gather_event_display_context__workflow_crawling_without_display_module():
```

Comment:
This test looks great. Don't change anything logic-wise, simply add `GIVEN`/`WHEN`/`THEN` comments throughout